PoseEstimation/poseEstimation_main.py

```diff
@@ -0,0 +1,71 @@
+import os, requests
+import sys
+import numpy as np
+import h5py
+import typing
+from pathlib import Path
+import matplotlib.pyplot as plt
+from tifffile import imwrite, TiffWriter, TiffFile
+
+sys.path.append("..")
+import utilities as utils
+import definitions as defs
+import poseEstimation_parameters as poseEst_params
+import poseEstimation_definitions as poseEst_defs
+
+import deeplabcut
+print("Imported DLC!", flush=True)
+
+# Import for PyInstaller
+from multiprocessing import freeze_support
+freeze_support()
+
+# Figure Style settings for notebook.
+import matplotlib as mpl
+mpl.rcParams.update({
+    'axes.spines.left': True,
+    'axes.spines.bottom': True,
+    'axes.spines.top': False,
+    'axes.spines.right': False,
+    'legend.frameon': False,
+    'figure.subplot.wspace': .01,
+    'figure.subplot.hspace': .01,
+    'figure.figsize': (18, 13),
+    'ytick.major.left': True,
+})
+jet = mpl.cm.get_cmap('jet')
+jet.set_bad(color='k')
+
+
+def main(poseEstimation_params: poseEst_params.PoseEstimationParameters):
+
+    """
+    DeepLabCut algorithm
+    """
+    #filePath: str = poseEstimation_params.paths[defs.Parameters.Path.FILEPATH]
+    #doricFile = h5py.File(filePath, 'r')
+
+    path_config_file: str = poseEstimation_params.paths[defs.Parameters.Path.TMP_DIR] + "/config.yaml"
```

Comment:
I feel like it would be better to create project and generate the config file here, and pass from danse only necessary info as parameters